src/DripsHub.sol

```diff
@@ -219,21 +222,24 @@ abstract contract DripsHub {
         view
         returns (uint128 collected, uint128 split)
     {
-        ReceiverState storage receiver = _dripsHubStorage().receiverStates[user];
+        bytes32 userHash = _hashUser(user);
```

Comment:
I think `userID` would be a better name. Also maybe `_calcUserID` instead of `_hashUser`.

Then it is clearer, that is an `ID`. Otherwise the reader might confuse it with all the other hashes we are dealing with.

Reply:
Agreed, fixed